lib/src/widgets/atomic/next_class.dart

```diff
@@ -64,7 +65,13 @@ class NextClass extends StatelessWidget {
 				page: Routes.schedule,
 				title: period == null
 					? "School is over"
-					: "${next ? 'Up next' : 'Right now'}: ${period!.getName(subject)}"
+					: "${next ? (Models.instance.schedule.periods) != null ? (
+					(Time.fromDateTime(DateTime.now()) <
+					Models.instance.schedule.periods![0].time.start) ?
+					'Second Period': 'Up Next'): "Up Next" : (Models.instance.schedule.periods)
+					!= null ? ((Time.fromDateTime(DateTime.now()) <
+					Models.instance.schedule.periods![0].time.start) ?
+					'First Period': 'Right Now'):'Right Now'}: ${period!.getName(subject)}"
```

Comment:
Why not just treat the first period as the current period and the 2nd period as the next period? In other words, treat "school hasn't started yet" as the first period since that's the info users need to see most. Remember that "school hasn't started yet" typically means a few minutes before the bell rings so it's not too weird to see "Right now" or "Up next"

Reply:
But I feel like everyone knows that school has started yet no? I think it might just make more sense to have "First Period" and "Second Period". Plus it looks pretty nice.